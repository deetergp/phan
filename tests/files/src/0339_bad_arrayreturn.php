<?php
/**
 * @return int[]
 */
function testarrayreturn() : array {
    return [42, "value", [2]];
}
