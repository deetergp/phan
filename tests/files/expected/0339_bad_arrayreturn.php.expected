%s:6 PhanTypeMismatchReturn Returning type int[][] but testarrayreturn() is declared to return int[]
%s:6 PhanTypeMismatchReturn Returning type string[] but testarrayreturn() is declared to return int[]
